models/booking.js

```diff
@@ -1,7 +1,20 @@
 const mongoose = require('mongoose');
```

Comment:
Let's play with ES6 a bit and destructure mongoose to take from it only stuff we want.
So write it like this: 
```js
const { model, Schema } = require('mongoose');
```
And then you just have
```js
const bookingSchema = new Schema({ /* Schema def */ });

const booking = model('Booking', bookingSchema);
```